packages/nextjs/components/example-ui/ContractInteraction.tsx

```diff
@@ -19,6 +20,45 @@ export const ContractInteraction = () => {
     },
   });
 
+  // Define the modal trigger button
```

Comment:
let's change this file when we'll have designs
https://github.com/scaffold-eth/scaffold-eth-2/issues/466